src/converter.cc

```diff
@@ -186,6 +186,9 @@ converter_test (bool verbose)
 {
     printf (" * converter: ");
 
+    ManageFtyLog::setInstanceFtylog("fty_metric_store_server");
```

Comment:
should be fty_metric_converter or something similar probably rather than "fty_metric_store_server"